packages/editor/src/bindings/post-meta.js

```diff
@@ -7,22 +7,43 @@ import { store as coreDataStore } from '@wordpress/core-data';
  * Internal dependencies
  */
 import { store as editorStore } from '../store';
+import { unlock } from '../lock-unlock';
+
+function getMetadata( registry, context ) {
```

Comment:
This name `getMetadata` is too generic even as an internal util.

Reply:
This function also could be documented. If I follow the code properly, the function returns the list of meta fields and their values.

Reply:
I can work on a new name and documentation in a follow-up PR among other improvements to this function.